lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java

```diff
@@ -52,7 +52,7 @@ private static QuizQuestion makeQuestion0() {
             AnswerChoice.B, "The monitor",
             AnswerChoice.C, "The keyboard",
             AnswerChoice.D, "The mouse"),
-        AnswerChoice.UNANSWERED); // Replace `UNANSWERED` with the correct answer.
+        AnswerChoice. A); // Replace `UNANSWERED` with the correct answer.
```

Comment:
Need to fix formatting here and below. Run `./gradlew spotlessApply" and then push your changes per the updated instructions.